src/modules/flyout/flyout.component.html

```diff
@@ -1,26 +1,24 @@
 <div
   class="sky-flyout"
   tabindex="-1"
-  (@flyoutState.done)="animationDone($event)"
-  [@flyoutState]="getAnimationState()"
   [attr.role]="config?.ariaRole"
   [attr.aria-describedby]="config?.ariaDescribedBy"
   [attr.aria-labelledby]="config?.ariaLabelledBy"
->
-  <div class="sky-flyout-header">
+  [ngClass]="{ 'sky-flyout-hidden': !isOpen && !isOpening }"
+  (@flyoutState.done)="animationDone($event)"
+  [@flyoutState]="getAnimationState()">
+  <div class="sky-flyout-header" #header>
```

Comment:
will `header` collide with any other template references?

Reply:
This is a good observation, and it probably wouldn't hurt to prefix template reference variables with something unique, but Angular does a great job of managing your context for you. The use of `header` here would only be problematic if we had an unrelated component property also named `header`, that was _also_ referenced in the template HTML. I'll go ahead and make the change, though, because it doesn't hurt to be more specific.

https://angular.io/guide/template-syntax#expression-context